Presentation.Web/Controllers/API/V1/UserController.cs

```diff
@@ -192,7 +191,7 @@ public override HttpResponseMessage Delete(int id, int organizationId = 0)
             var uuid = _identityResolver.ResolveUuid<User>(id);
             return uuid.IsNone
                 ? NotFound()
-                : _userService.DeleteUserFromKitos(uuid.Value).Match(FromOperationError, Ok);
+                : _userService.InitiateUserDeletion(uuid.Value, organizationId).Match(FromOperationError, Ok);
```

Comment:
move this endpoint to a new controller with the api/v1 etc route structure so you dont have that "fake" org id parameter that forced you into the mess in the service

Reply:
once the new controller is created, annotate this with [NoAction] and implement by throwing a notsupportedexception (see other places where we use ther noAction attr for reference)

Reply:
Also once migrated, add the integration test we talked about.
Scenario:

Test with both 
- "Global admin user which is NOT local admin in any of the tested orgs"
- Local admin user which is local admin in both orgs

Create two orgs
Assign a standard user to both orgs
Call the endpoint in org 1
Verify that the user no longer has roles in org1 but that the user has not been deleted
Call the endpoint in org 2
Verify that the user no longer has roles in org2 AND has been "deleted"

Repeat for both auth types